README.md

```diff
@@ -1,6 +1,6 @@
 
-# CoL Data Package (COLDP) Specification
-The recommended exchange format for data to and from the CoL Clearinghouse
+# COL Data Package (COLDP) Specification
+The recommended exchange format for data to and from the COL ChecklistBank
```

Comment:
I am no native speaker, but when changing from clearinghouse to checklistbank I would also remove the `the` and just call it _from COL ChecklistBank_

Reply:
It is ok to drop 'the'